backend/dataall/modules/dashboards/db/dashboard_models.py

```diff
@@ -37,11 +37,11 @@ class Dashboard(Resource, Base):
     userRoleForDashboard = query_expression()
 
     @classmethod
-    def get_uri(cls):
-        return cls.dashboardUri
+    def uri(self):
+        return self.dashboardUri
```

Comment:
`@classmethod` doesn't have access to `self`, it is invoked as `Dashboard.uri` thus it doesn't know the instance.